tests/unit/lms/views/admin/application_instance_test.py

```diff
@@ -61,6 +61,44 @@ def test_instance_with_duplicate(self, views, application_instance_service):
 
         assert response.status_code == 400
 
+    def test_move_application_instance_org(
+        self,
+        views,
+        pyramid_request,
+        application_instance,
+        application_instance_service,
+    ):
+        application_instance_service.get_by_id.return_value = application_instance
+        pyramid_request.params["org_public_id"] = "PUBLIC_ID"
+
```

Comment:
Might this benefit from an explicit test case that says we don't lose the settings? Or is that an example of testing one of the infinite number of things we don't expect to happen.

I'm more inclined towards it as it's a bug that actually happened.

Reply:
Not sure how the test could look:

- The update AI view still has the same issue, if you don't pass the settings it will update them all.

- The new view doesn't get near the settings. We could test that it doesn't indeed change them.

I'll have a think and maybe propose an alternative fix and test. I'll merge this now to avoid further updating data in production.